src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -55,16 +56,14 @@ const Stream = createReactClass({
     let sort = 'sort' in currentQuery ? currentQuery.sort : DEFAULT_SORT;
 
     let hasQuery = 'query' in currentQuery;
-    let validStatsPeriods = new Set(['14d', '24h']);
-    let statsPeriod =
-      validStatsPeriods.has(currentQuery.statsPeriod) || DEFAULT_STATS_PERIOD;
```

Comment:
i guess it could have been `true` here too in the old version, but it seems like you'd want a ternary